convert_to_excel.py

```diff
@@ -56,53 +62,70 @@
 from openpyxl.styles import Font
 from openpyxl.styles import Alignment
 from openpyxl.styles import PatternFill
-
-print('Start\n')  # Only for testing purposes---.
-
+import sys
+import os
 ##############################################################################
-""" Variables, lists, etc. """
 
 
-column_names = ['Task', 'Set Part', 'Parent Build', 'Start Date', 'End Date']
-corresponding_keys = ['content', 'entity', 'sg_parent_build',
-                      'start_date', 'due_date']
-
-imported_data = 'test_data.pkl'
-workbook_title = 'converted_data.xlsx'
-worksheet_title = 'Formatted'
-
 
 ##############################################################################
-""" 1. Unpickle file and place into a list. """
-
-# Convert pickled data into a list.
-unpickled_data = open(imported_data, 'rb')
-data = pickle.load(unpickled_data)
-# print(data)  # [works] This is a list, not a dict.
-
-# Check type of the converted data.
-# t = type(data)  # Only for testing purposes---.
-# print('type = {}\n'.format(t))  # Only for testing purposes---.
+""" 1. Defining the data class. """
+
+
+class CapturedData:
+    def __init__(self, object_name, task_name, set_part,
+                 parent_build, start_date,
+                 end_date):
+
+        self.object_name = object_name
+        self.task_name = task_name
+        self.set_part = set_part
+        self.parent_build = parent_build
+        self.start_date = start_date
+        self.end_date = end_date
+
+    def print_status(self):
+        print(F'object_name = {self.object_name}')
+        print(F'task_name = {self.task_name}')
+        print(F'set_part = {self.set_part}')
+        print(F'parent_build = {self.parent_build}')
+        print(F'start_date = {self.start_date}')
+        print(F'end_date = {self.end_date}')
+
+    # These two functions are only used if the data parsed, does not work.
+    def set_start_date(self):
+        date_format = "%Y-%m-%d"
+        self.start_date = datetime.strptime(self.start_date, date_format)
+        self.start_date = self.start_date.strftime("%Y-%m-%d")
+
+    def set_end_date(self):
+        date_format = "%Y-%m-%d"
+        self.end_date = datetime.strptime(self.end_date, date_format)
+        self.end_date = self.end_date.strftime("%Y-%m-%d")
 
 ##############################################################################
-""" 2. Sort data by earliest start date. """
+""" 2. Create an excel file. """
 
-# Find size - how many lists are inside this list.
-list_size = len(data)
-print('Size of list = {}\n'.format(list_size))
+def check_file_type(file_to_check):
```

Comment:
I would remove this function entirely, across all of the tests you have used it in. `check_file_type` is re-used to mean 3 different things in 3 different exercises here and in the other PR. And the function doesn't provide any real value. Rather than writing:

`if check_file_type(my_file):`
you could write
`if my_file.endswith('.pkl'):`

And it saves defining this 5 line function and makes the resulting code more legible.

I would give the same feedback for the functions `remove_brackets` and `remove_quotes`. It's more legible to call `.strip()` in your code rather than writing a function that calls `.strip()`